minvws/nl-kat-coordination
# Fixes in Report Overview

### Changes
This PR makes the following changes to the Report Overview:
- Fix the breadcrumbs
- Change the page description text on the Scheduled Reports page
- Change the page description text on the Reports History page
- If scheduled_reports_table or reports_history_table is empty, show an empty state text (instead of an empty table)
- Remove the "Subreport Name" column in the Scheduled Reports table

### Demo
![afbeelding](https://github.com/user-attachments/assets/91a5ac62-7258-4fa2-a9f1-b1ff681db796)
![afbeelding](https://github.com/user-attachments/assets/2711d693-8a55-49de-820a-f3ccd2f260fe)
![afbeelding](https://github.com/user-attachments/assets/d7e9d49a-c497-4bbb-9afd-be7eb1e8d796)


### QA notes
- Go to the Report page. 
- Check the two tabs "History" and "Scheduled". Check the intro text and the empty state text.
- Create a report without schedule and a scheduled report
- Check the two tabs again. The table should be visible (no empty state text) and counter should work


---

### Code Checklist







---

## Checklist for code reviewers:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_code.md) into your comment.

---

## Checklist for QA:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_qa.md) into your comment.
